db/config.cc

```diff
@@ -351,9 +351,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     /**
     * @Group Ungrouped properties
     */
-    , background_writer_scheduling_quota(this, "background_writer_scheduling_quota", value_status::Unused, 1.0,
+    , background_writer_scheduling_quota(this, "background_writer_scheduling_quota", value_status::Deprecated, 1.0,
         "max cpu usage ratio (between 0 and 1) for compaction process. Not intended for setting in normal operations. Setting it to 1 or higher will disable it, recommended operational setting is 0.5.")
-    , auto_adjust_flush_quota(this, "auto_adjust_flush_quota", value_status::Unused, false,
+    , auto_adjust_flush_quota(this, "auto_adjust_flush_quota", value_status::Deprecated, false,
```

Comment:
There's also the guardrails replication strategy option that we marked as Unused recently, but it should also become Deprecated

Reply:
Right! Thanks, fixed.